<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceTest</class>
 <widget class="QDialog" name="DeviceTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备测试</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnCamera">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>摄像头</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCamera">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMic">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>麦克风</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbMic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPlayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlayer">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>扬声器</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="VideoRender" name="widRender" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>347</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QSlider" name="hsPlayerVol">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>245</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbPlayer">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>245</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>扬声器音量</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPlayerVol">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>245</y>
     <width>38</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSlider" name="hsMicVol">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbMicVol">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>38</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbMic">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>麦克风增强</string>
   </property>
  </widget>
  <widget class="QSlider" name="vsSkinSmooth">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>40</y>
     <width>25</width>
     <height>200</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbSkinSmooth">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>245</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>美颜</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbSkinSmooth">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>10</y>
     <width>25</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbSkinWhite">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>10</y>
     <width>25</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSlider" name="vsSkinWhite">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>40</y>
     <width>25</width>
     <height>200</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbSkinWhite">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>245</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>美白</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoRender</class>
   <extends>QWidget</extends>
   <header>videorender.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
